package.json

```diff
@@ -31,6 +31,9 @@
     "he": "^1.1.1",
     "marked": "^0.3.12"
   },
+  "jest": {
+    "testURL": "http://localhost/"
+  },
```

Comment:
Instead you can upgrade to jest 23.5.0 which fixes this issue.
https://github.com/facebook/jest/pull/6792

Reply:
Ah, nice, they fixed it! Thanks! Rebased now :)